### PR TITLE
Add sxhkd-mode-indentation-length var to set indentation length

### PR DESCRIPTION
**Pull request by jriboux**
Added `sxhkd-mode-indentation-length` var to set indentation length.

Renamed `sxhkd-mode-indentation-length` function to `sxhkd-mode-current-indentation-length`.

### CODE REVIEW

**Comment by xFA25E** [comment]
Hi! Thanks for your work!

File: sxhkd-mode.el [hunk 1]
```diff
@@ -108,8 +111,8 @@ Can be t, nil or ask"
       (if (looking-at-p (rx (or "#" (and bol eol))))
           0
         (end-of-line)
-        (forward-char -1)
-        (if (looking-at-p (rx "\\")) 0 1)))))
+        (forward-char (* -1 sxhkd-mode-indentation-length))
```
Comment (xFA25E) [inline sxhkd-mode.el]: Could you please change this line to `(- sxhkd-mode-indentation-length)`? Thanks!

File: sxhkd-mode.el [hunk 2]
```diff
@@ -44,6 +44,9 @@ Can be t, nil or ask"
                 (const :tag "Ask" ask))
   :group 'sxhkd-mode)
 
+(defvar sxhkd-mode-indentation-length 2
+  "Indentation length")
```
Comment (xFA25E) [inline sxhkd-mode.el]: This one should be defcustom:
```
(defcustom sxhkd-mode-indentation-length 2
  "Indentation length."
  :type 'integer
  :group 'sxhkd-mode)
```

File: sxhkd-mode.el (hunk 1 above)
Comment (jriboux) [reply sxhkd-mode.el]: Thanks for the review, it is fixed in last commit.

File: sxhkd-mode.el (hunk 2 above)
Comment (jriboux) [reply sxhkd-mode.el]: Thanks for the review, it is fixed in last commit.

**Pull request merged**